app/controllers/v0/health_care_applications_v2_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module V0
+  class HealthCareApplicationsV2Controller < ApplicationController
+    include HcaValidate
+
+    def create
+      authenticate_token
+
+      form = JSON.parse(params[:form])
+      validate!(form)
+
+      health_care_application = HealthCareApplication.create!
+
+      HCA::SubmissionJob.perform_async(current_user&.uuid, form, health_care_application.id)
```

Comment:
I know are requirements keep changing, but I'm afraid we need to change this to submit forms without an email address synchronously.  See [Ryan's comment.](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10202#issuecomment-387088318)

Reply:
yea, the original HCA route still submits synchronously and this route is the async one

Reply:
I see.  What if we instead put all HCA responsibility to the old controller?
* have `HealthCareApplicationsController#create` handle sync and async submissions, and move `HealthCareApplicationsV2Controller#show` to `HealthCareApplicationsController`
* have the responses from async processing contain at least the same attributes as synchronous processing (we would just have to add `success: true` to `HealthCareApplicationSerializer`), and add an attribute to indicate if the HCA was processed synchronously or asynchronously

Benefits:
* frontend does not have to change any existing URLs.  They post all HCAs to the same URL as they do now.  They may display different messages to the user depending on what is returned.
* we don't have to worry about stale frontend code since the same URL is used and returns an object the frontend already understands.
* the API does not contain two controllers for the same resource, which may cause confusion in the future when trying to decide which controller should be responsible for new tasks.

Reply:
@rlbaker What do you think?  Another opinion would be appreciated.

Reply:
If it were me sitting down to implement it today, keeping the new HCA work as separate from the current implementation until it's deployed and proven seems like a better approach so that we don't accidentally break any existing HCA work. Rails makes it fairly simple for us to alias routes in the future if we want to do a switch-over to a unified route in the future.

I definitely see the value of your approach, @jholton, and it's very well considered, but I wonder if it would be better to first implement and validate this work as a separate component and merge/alias them later.

Happy to discuss further however!